packages/fxa-auth-server/docs/swagger/misc-api.ts

```diff
@@ -35,6 +35,11 @@ const ACCOUNT_SESSIONS_LOCATIONS_GET = {
   notes: ['🔒 Authenticated with support panel secret'],
 };
 
+const ACCOUNT_SET_PASSWORD_POST = {
+  ...TAGS_MISC,
```

Comment:
I noticed the `/account/finish_setup` route was tagged as Misc rather than Account. Is that correct? Certainly these routes are account routes, though they are related specifically to subscription account flows I suppose...

Reply:
Ben: It's possible some of these aren't quite in the right place from the initial Swagger integration.

@xlisachan : Hello! I plan to update the tags for these account routes (/account/finish_setup, /account/set_password, maybe some other `/account/...` ones that are tagged Misc) to be under Account instead of Misc. Please let me know if you see any problems with this.